www/src/views/community/index.js

```diff
@@ -143,7 +143,7 @@ class CommunityView extends Component {
                     <div
                       css={{
                         margin: `0 0 ${rhythm(1 / 8)}`,
-                        color: colors.gray.bright,
+                        color: colors.gray.calm,
```

Comment:
Guessing the contrast was too low on these?

Reply:
Yes! It now is on the AA level.